ui/admin/app/styles/app.scss

```diff
@@ -953,9 +953,29 @@
 
 // workers
 .workers {
-  .rose-dropdown-content {
-    max-height: 300px;
-    overflow: scroll;
+  .workers-filtering {
+    display: flex;
+    margin-bottom: 1rem;
+
+    .toolbar-refresher {
+      margin-left: auto;
+    }
+
+    .hds-dropdown-list-item--variant-generic {
+      padding: 0.5rem 1rem;
+      text-transform: uppercase;
+      color: var(--ui-gray)
```

Comment:
Could you have just used the `@color` arg on the HDS components themselves?

Reply:
I wanted the icon and the text color to match what's currently in prod, so I moved the color to the class instead of using the `@color` arg

Reply:
Is the `@color='faint'` not close enough? I think it's probably better to use the HDS color scheme where we can at this point

Reply:
so are we ok with having the icon in black color? if I use the @color args we either leave the icons black or handle this icon's color in the css.   
<img width="245" alt="Screenshot 2025-05-12 at 3 51 59 PM" src="https://github.com/user-attachments/assets/7ba83e7b-6a36-4f04-9df4-a4c6a6840d8c" />

Reply:
Why not make the icon [color](https://helios.hashicorp.design/components/icon?tab=code#color) also the same?

Reply:
oops I forgot we could do that for icons!